app-modules/student-data-model/routes/api.php

```diff
@@ -84,5 +86,8 @@
                     Route::patch('/{studentPhoneNumber}', UpdateStudentPhoneNumberController::class)->name('update');
                     Route::delete('/{studentPhoneNumber}', StudentPhoneNumbersDeleteStudentPhoneNumberController::class)->name('delete');
                 });
+
+            Route::get('{student}/enrollments', ListStudentEnrollmentsController::class)->name('enrollments.index');
+            Route::put('{student}/enrollments', StudentEnrollmentsController::class)->name('enrollments.put');
```

Comment:
This could be put into a group so the route and names can be shared like the email addresses and phone numbers routes above.